tests/linearGaussian_snpe_test.py

```diff
@@ -144,13 +143,13 @@ def simulator(theta):
         assert ((map_ - ones(num_dim)) ** 2).sum() < 0.5
 
 
-@pytest.mark.slow
+# @pytest.mark.slow
 @pytest.mark.parametrize(
     "density_estimator",
     ["made", "mdn", "maf", "maf_rqs", "nsf", "zuko_maf", "zuko_nsf"],
 )
 def test_density_estimators_on_linearGaussian(density_estimator):
-    """Test NPE with different density estimators on linear Gaussian example."""
+    """Test NPE B/C with different density estimators on linear Gaussian example."""
```

Comment:
I would run this test with both `NPE_B` and `NPE_C` (they should give the same thing though)